tests/integration/create_product.t

```diff
@@ -2,16 +2,16 @@
 
 use ProductOpener::PerlStandards;
 
-use Test;
+use Test::More;
 use ProductOpener::APITest qw/:all/;
 use ProductOpener::Test qw/:all/;
 
 remove_all_products();
 wait_dynamic_front();
 
-my $ua = new_client();
+my $admin_ua = new_client();
```

Comment:
if we keep test, it's not an admin but a simple contributor ;-)

So $contrib_ua might be more adapted.